pages/subjects/create-a-subject.json

```
@@ -0,0 +1,7 @@
+{
+    "id": 168,
+    "name": "Statistic",
```

```suggestion
    "name": "Statistics",
```

Sorry but my brain x